Failing to determine X boundaries during diffraction focussing for load tests

Changes in [this PR](https://github.com/neutrons/mantid_total_scattering/pull/86) to fix wavelength / TOF min max now have failing tests during `AlignAndFocusPowderFromFiles` calls during `DiffractionFocussing` in `tests/file_handling/test_load.py`

Output from pipeline https://github.com/neutrons/mantid_total_scattering/runs/2420423219?check_suite_focus=true

```
   E           RuntimeError: AlignAndFocusPowder-v1: Fail to determine X boundaries for group:4
  E           The boundaries are (Xmin,Xmax):1524.22 1522.61
  E             at line 603 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  E             caused by line 702 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  E             caused by line 428 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  E             caused by line 1049 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/lib/python3.7/site-packages/mantid/simpleapi.py'
```
and
```
AlignAndFocusPowder-[Notice] AlignAndFocusPowder started (child)
  Error: trument-[Error] MaskInstrument is deprecated (on 2020-07-16). Use MaskDetectors instead.
  Error: tectors-[Error] AlignDetectors is deprecated (on 2021-01-04). Use ConvertUnits instead.
  MaskDetectors-[Notice] Masking using workspace indicies
  Error: tionFocussing-[Error] Error in execution of algorithm DiffractionFocussing:
  Error: tionFocussing-[Error] Fail to determine X boundaries for group:4
  Error: tionFocussing-[Error] The boundaries are (Xmin,Xmax):1524.22 1522.61
  Error: dFocusPowder-[Error] Error in execution of algorithm AlignAndFocusPowder:
  Error: dFocusPowder-[Error] Fail to determine X boundaries for group:4
  Error: dFocusPowder-[Error] The boundaries are (Xmin,Xmax):1524.22 1522.61
  Error: dFocusPowderFromFiles-[Error] Error in execution of algorithm AlignAndFocusPowderFromFiles:
  Error: dFocusPowderFromFiles-[Error] AlignAndFocusPowder-v1: Fail to determine X boundaries for group:4
  Error: dFocusPowderFromFiles-[Error] The boundaries are (Xmin,Xmax):1524.22 1522.61
  Error: dFocusPowderFromFiles-[Error]   at line 603 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  Error: dFocusPowderFromFiles-[Error]   caused by line 702 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  Error: dFocusPowderFromFiles-[Error]   caused by line 428 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/plugins/python/algorithms/AlignAndFocusPowderFromFiles.py'
  Error: dFocusPowderFromFiles-[Error]   caused by line 1049 in '/home/runner/work/mantid_total_scattering/mantid_total_scattering/.tox/py37/lib/python3.7/site-packages/mantid/simpleapi.py'
```

--- a/.github/workflows/actions.yml
+++ b/.github/workflows/actions.yml
@@ -9,6 +9,9 @@ on:
       - '*'
     tags:
       - 'v*'
+  pull_request:
+    types: [ opened, edited, reopened, review_requested ]
+    branches: [ master ] 
 
 jobs:
   linux:

--- a/.github/workflows/actions.yml
+++ b/.github/workflows/actions.yml
@@ -10,8 +10,8 @@ on:
     tags:
       - 'v*'
   pull_request:
-    types: [ opened, edited, reopened, review_requested ]
-    branches: [ master ] 
+    branches:
+      - '*'
 
 jobs:
   linux:

--- a/total_scattering/file_handling/load.py
+++ b/total_scattering/file_handling/load.py
@@ -152,7 +152,7 @@ def create_absorption_wksp(filename, abs_method, geometry, material,
                 "lambda",
                 "skf12.lambda",
                 "BL1B:Det:TH:BL:Lambda",
-                "freq"]
+                "frequency"]
 
             for logname_wl in wl_lognames:
                 run = abs_input.run()

